CMakeLists.txt

```diff
@@ -92,6 +92,8 @@ elseif(CMAKE_SYSTEM_NAME STREQUAL "Darwin")
   add_compile_definitions(__LINUX__ LINUX _MAX_PATH=260 _MAX_FNAME=256 _REENRANT MACOSX=1 _USE_OGL_ACTIVE_TEXTURES)
   set(PLATFORM_INCLUDES "lib/linux" ${SDL2_INCLUDE_DIR} ${SDL2_INCLUDE_DIRS})
 elseif(CMAKE_SYSTEM_NAME STREQUAL "Windows")
+  add_compile_definitions(WIN32_LEAN_AND_MEAN NOMINMAX NODRAWTEXT #[[NOGDI]] NOBITMAP NOMCX NOSERVICE NOHELP)
```

Comment:
So every `WIN32_LEAN_AND_MEAN` define in the code can go ? There are a dozen scattered around

Reply:
Yeah, I can remove all of them, will do